packages/fast-components-styles-msft/README.md

```diff
@@ -119,6 +119,293 @@ const StyleSheet = {
 
 These utility functions help you write clean and manageable style sheets.
 
+- [Acrylic](#acrylic)
+    - [applyAcrylicMaterial](#applyacrylicmaterial)
+- [Border](#border)
+    - [applyCornerRadius](#applycornerradius)
+    - [applyFloatingCornerRadius](#applyfloatingcornerradius)
+    - [applyFocusPlaceholderBorder](#applyfocusplaceholderborder)
+- [Cursor](#cursor)
+    - [applyCursorDefault](#applycursordefault)
+    - [applyCursorDisabled](#applycursordisabled)
+    - [applyCursorPointer](#applycursorpointer)
+- [Density](#density)
+    - [DensityCategory](#densitycategory)
+    - [height](#height)
+    - [heightNumber](#heightnumber)
+    - [densityCategorySwitch](#densitycategoryswitch)
+    - [horizontalSpacing](#horizontalspacing)
+    - [horizontalSpacingNumber](#horizontalspacingnumber)
+    - [glyphSize](#glyphsize)
+- [Disabled](#disabled)
+    - [applyDisabledState](#applydisabledstate)
+- [Elevation](#elevation)
+    - [ElevationMultiplier](#elevationmultiplier)
+    - [applyElevation](#applyelevation)
+- [Fonts](#fonts)
+    - [applyFontWeightLight](#applyfontweightlight)
+    - [applyFontWeightSemiLight](#applyfontweightsemilight)
+    - [applyFontWeightNormal](#applyfontweightnormal)
+    - [applyFontWeightSemiBold](#applyfontweightsemibold)
+    - [applyFontWeightBold](#applyfontweightbold)
+- [Typography](#typography)
+    - [applyScaledTypeRamp](#applyscaledtyperamp)
+    - [applyTypeRamp](#applytyperamp)
+    - [applyScaledFontSize](#applyscaledfontsize)
+    - [applyScaledLineHeight](#applyscaledlineheight)
+    - [applyFontSize](#applyfontsize)
+    - [applyLineHeight](#applylineheight)
+
+### Acrylic
+
+#### applyAcrylicMaterial
+
+Apply a Microsoft implementation of acrylic to a styled element.

```

Comment:
can you describe what acrylic is/should look like, outside of understanding the design system this is ambiguous. Perhaps something like "...acrylic, a blurred semi-transparent surface, to a styled element."

Reply:
Added above; pulled almost directly from the MSFT docs